webpack/assets/javascripts/react_app/components/Truncate/index.js

```diff
@@ -0,0 +1,3 @@
+import Truncate from './Truncate';
```

Comment:
index file isn't needed here, we use it as a convention for redux connected component